src/build.js

```diff
@@ -30,7 +34,8 @@ module.exports = function build(markdown) {
   return `
 ${doImports}
 
-module.exports = function() {
+export const attributes = ${JSON.stringify(camelize(frontMatterAttributes))};
+export default function() {
   return (
     <div>
```

Comment:
Im pretty sure if you use commonJS modules you can't use es6 modules as it causes issues with certain parsers, it is best to stick with one